src/patients/list/Patients.tsx

```diff
@@ -20,6 +20,18 @@ import useAddBreadcrumbs from '../../breadcrumbs/useAddBreadcrumbs'
 
 const breadcrumbs = [{ i18nKey: 'patients.label', location: '/patients' }]
 
+function useDebounce(value: any, delayInMilliseconds: number) {
```

Comment:
I think that we should extract this to its own hook! I can see this being useful in other places in the application.